### PR TITLE
dotCMS/core#22724 Create custom tool from Content Type

### LINKED ISSUE #22724: Create custom tool from Content Type
**Is your feature request related to a problem? Please describe.**
As a user, I want to be able to add content types to the navigation as easy and fast as possible while creating or edit a content type.

**Describe the solution you'd like**
Add a "Add to menu" button to the toolbar.

Screen Shot 2022-08-09 at 16 13 03

https://user-images.githubusercontent.com/751424/220363003-75e491fe-55b9-435d-b5eb-e2c30ad066b9.mp4

And show the same dialog we use from the listing.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* show Add to Menu Dialog on Content Type Edit Screen 

### Checklist
- [x] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

**Comment by zulqarnainvd** [comment]
[screen-capture (8).webm](https://user-images.githubusercontent.com/113915849/220394833-39b53c03-95e5-4abf-9d01-681f2ad0a9bd.webm)

**Comment by zulqarnainvd** [comment]
old

https://user-images.githubusercontent.com/113915849/220395040-c9e38dcf-0242-437f-9c8e-979ab5af95be.mp4

**Review by oidacra — APPROVED** [review]

File: core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html [hunk 1]
```diff
@@ -44,6 +44,14 @@ <h4 (click)="editInlineActivate($event)">{{ contentType.name }}</h4>
         </div>
     </div>
     <div class="main-toolbar-right">
+        <button
+            class="p-button-secondary content-type__add-to-menu"
+            id="add-to-menu-button"
+            (click)="addContentInMenu()"
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html:50]: click functionality need to be tested

File: core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html [hunk 2]
```diff
@@ -115,3 +123,8 @@ <h4 (click)="editInlineActivate($event)">{{ contentType.name }}</h4>
         </ng-template>
     </p-tabPanel>
 </p-tabView>
+<dot-add-to-menu
+    *ngIf="addToMenuContentType"
+    [contentType]="contentType"
+    (cancel)="addToMenuContentType = false"
```
Comment (alfredo-dotcms) [inline core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html:129]: CANCEL functionality need to be tested

**Review by fmontes — COMMENTED** [review]
Check Alfredo's comments.

File: core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html (hunk 1 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html:50]: Noted

File: core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html (hunk 2 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html:129]: Noted

File: core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html (hunk 1 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html:50]: updated

File: core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html (hunk 2 above)
Comment (zulqarnainvd) [reply core-web/apps/dotcms-ui/src/app/portlets/shared/dot-content-types-edit/components/layout/content-types-layout.component.html:129]: updated

**Review by alfredo-dotcms — APPROVED** [review]

**Pull request merged**